openmctStaticServer/plugins/conf/dictionaryMySavedPanels.json

```diff
@@ -35,6 +35,14 @@
       "telemetryEntries": [
         {"namespace": "myPanel200", "key": "root"}
       ]
+    },
+    {
+      "name": "Logging Tables",
```

Comment:
Just a small comment. The name is a bit generic

Reply:
Indeed, "Process Yarp Text Logging Tables" maybe?

Reply:
Sorry, I misunderstood. That's the name of the folder. That's perfectly fine as it is